src/ManageCourses.Api/Services/Users/UserService.cs

```diff
@@ -59,6 +59,7 @@ public async Task UserSignedInAsync(string accessToken, JsonUserDetails userDeta
 
             _context.Save();
             SendWelcomeEmail(mcUser);
+            return mcUser;
```

Comment:
I think this was the original dawn of time approach that you disliked right?

Reply:
Probably :-)

Reply:
We can take it back out again when we switch to Id as a FK